internal/cli/cluster/cluster_chart.go

```diff
@@ -201,7 +200,8 @@ func (c *ChartInfo) buildClusterSchema() error {
 
 func (c *ChartInfo) buildClusterDef() error {
 	cht := c.Chart
-	clusterFilePath := filepath.Join(templatesDir, clusterFile)
+	// use filepath.Join(templatesDir, clusterFile) will make the clusterFilePath in windows
+	clusterFilePath := templatesDir + "/" + clusterFile
```

Comment:
We use embed FS to read chart's tgz files. In embed FS, the file path format is compatible with Linux and does not change with the operating system. Therefore, we cannot use filepath.Join to generate different path formats for different systems, instead, we need to use a path format that is the same as Linux.